Ukhu/Banka
# #165116610 Set up Coveralls integration

#### What does this PR do?
Integrate test coverage reporting

#### Description of Task to be completed?
- Install nyc
- Install coveralls
- Add a script to the package.json to run the test script and pipe the report to coveralls
- Configure travis to send coverage reports to coveralls.io

#### How should this be manually tested?
Check the README page of the repo to see test coverage badge, which indicates successful integration

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165116610

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
